Sources/Sentry/SentryRateLimitCategoryMapper.m

```diff
@@ -15,4 +15,14 @@ + (NSString *_Nonnull)mapEventTypeToCategory:(NSString *)eventType {
     return SentryRateLimitCategoryError;
 }
 
++ (NSString *_Nonnull)mapEnvelopeItemTypeToCategory:(NSString *)itemType {
+    if ([itemType isEqualToString:SentryEnvelopeItemTypeEvent]) {
+        return SentryRateLimitCategoryError;
+    }
+    if ([itemType isEqualToString:SentryEnvelopeItemTypeSession]) {
```

Comment:
l: Isn't this a 1 to 1 match which is covered by the fallback behavior?
What's the motivation of having this mapping here but not other types?

Reply:
It put it there to explicitly state that we are sure about this mapping. The others just use default because we don't know how to handle them.